File: solutions/java/darts/1/src/main/java/Darts.java
import java.lang.Math;

    
class Darts {
    int score(double xOfDart, double yOfDart) {
        double target = Math.sqrt(Math.pow(xOfDart, 2) + Math.pow(yOfDart, 2)); 
        if (target <= 1)
        {
            return 10;
        }
        if (target > 1 && target <= 5)
        {
            return 5;
        }
        if (target > 5 && target <= 10)
        {
            return 1;
        }
        else
        {
            return 0;
        }
    }
}

File: solutions/javascript/darts/1/darts.js
 export const score = (x, y) => {
   const landingPoint = Math.sqrt((x**2) + (y**2));
   
   return (landingPoint > 10) ? 0 
     : (landingPoint > 5 && landingPoint <= 10) ? 1
     : (landingPoint > 1 && landingPoint <= 5) ? 5
     : 10;
 } 
